Mjml.Net/Components/Body/TextComponent.cs

```diff
@@ -0,0 +1,102 @@
+                ["padding-top"] = AttributeTypes.PixelsOrPercent,
+                ["text-decoration"] = AttributeTypes.String,
+                ["text-transform"] = AttributeTypes.String,
+            };
+
+        public override Attributes? DefaultAttributes { get; } =
+            new Attributes
+            {
+                ["align"] = "left",
+                ["color"] = "#000000",
+                ["font-family"] = "Ubuntu, Helvetica, Arial, sans-serif",
+                ["font-size"] = "13px",
+                ["font-style"] = "normal",
+                ["letter-spacing"] = "none",
+                ["line-height"] = "1",
+                ["padding"] = "10px 25px",
+            };
+
+        public override void Render(IHtmlRenderer renderer, INode node)
+        {
+            var height = node.GetAttribute("height");
+
+            if (string.IsNullOrEmpty(height))
+            {
+                RenderTextContent(renderer, node);
+            }
+            else
+            {
+                renderer.Content(ConditionalTags.StartConditional);
+                renderer.ElementStart("table")
+                    .Attr("role", "presentation")
+                    .Attr("border", "0")
+                    .Attr("cellpadding", "0")
+                    .Attr("cellspacing", "0");
+
+                renderer.ElementStart("tr");
+                renderer.ElementStart("td")
+                    .Attr("height", height)
+                    .Style("vertical-align", "top")
+                    .Style("height", height);
+                renderer.Content(ConditionalTags.EndConditional);
+
+                RenderTextContent(renderer, node);
+
+                renderer.Content(ConditionalTags.StartConditional);
+                renderer.ElementEnd("td");
+                renderer.ElementEnd("tr");
+                renderer.ElementEnd("table");
+                renderer.Content(ConditionalTags.EndConditional);
+            }
+        }
+
+        private static void RenderTextContent(IHtmlRenderer renderer, INode node)
+        {
+            renderer.ElementStart("div")
+                .Style("font-family", node.GetAttribute("font-family"))
+                .Style("font-size", node.GetAttribute("font-size"))
+                .Style("font-style", node.GetAttribute("font-style"))
+                .Style("font-weight", node.GetAttribute("font-weight"))
+                .Style("letter-spacing", node.GetAttribute("letter-spacing"))
+                .Style("line-height", node.GetAttribute("line-height"))
+                .Style("text-align", node.GetAttribute("align"))
+                .Style("text-decoration", node.GetAttribute("text-decoration"))
+                .Style("text-transform", node.GetAttribute("text-transform"))
+                .Style("color", node.GetAttribute("color"))
+                .Style("height", node.GetAttribute("height"));
+
+            renderer.RenderChildren(new ChildOptions
```

Comment:
What happens with ReadOuterXml()?

Reply:
I had to make a few changes to the context to fix some bugs. I would just focus on the update to the code generator and then have a look to this again. Perhaps the world looks different now.

Reply:
In general I think the RawXML should render everything, we would have the same problem in other places as well, I guess. For example mj-raw could have the same issue.

Reply:
> In general I think the RawXML should render everything, we would have the same problem in other places as well, I guess. For example mj-raw could have the same issue.

Exactly, early testing shows that `ReadOuterXml()` works. That's a bit of a confusing name :D I'll write some more unit tests. Agreed, this needs to be re-usable with minimal impact to render speed. I think maybe creating new function called `GetContentRaw()` would be more suitable. Otherwise, we'll be doing the conditional check for every `GetContentCall()`.